src/main/java/ca/uhn/fhir/jpa/starter/HapiProperties.java

```diff
@@ -14,6 +14,7 @@
     static final String ALLOW_EXTERNAL_REFERENCES = "allow_external_references";
     static final String ALLOW_MULTIPLE_DELETE = "allow_multiple_delete";
     static final String ALLOW_PLACEHOLDER_REFERENCES = "allow_placeholder_references";
+    static final String CACHE_TIMEOUT = "cache_timeout";
```

Comment:
Could this be called `search_result_cache_timeout_millis` just to make it a bit more explicit what it is?

Reply:
totally agree! I changed it to the same name as the config dao so `reuse_cached_search_results_millis` if that's cool and rebased so no more conflicts from master. Thanks!